test/FileService.Validation.UnitTests/FileRequestValidatorTests.cs

```diff
@@ -1,77 +1,78 @@
-﻿using FluentValidation;
-using FluentValidation.TestHelper;
+﻿using FluentValidation.TestHelper;
 using LT.DigitalOffice.FileService.Models.Dto.Models;
+using LT.DigitalOffice.FileService.Models.Dto.Requests;
+using LT.DigitalOffice.FileService.Validation.Interfaces;
 using NUnit.Framework;
 
 namespace LT.DigitalOffice.FileService.Validation.UnitTests
 {
     public class AddNewFileRequestValidatorTests
     {
```

Comment:
File name and class name have a difference

Reply:
Fixed